src/pg.py

```diff
@@ -736,24 +736,27 @@ def stop_postgresql(self, timeout=60):
                 logging.warning(line.rstrip())
         return self._cmd_manager.stop_postgresql(timeout, self.pgdata)
 
-    def replication_slots(self, action, slots):
-        """
-        Perform replication slots action (create/drop)
-        """
+    def get_replication_slots(self):
```

Comment:
Why is this function needed?
It has no added value at all.
If you want to make `_get_replication_slots()` usable outside class, just rename it.